### PR TITLE
generated galleryApplications cmdlets

### PR DESCRIPTION
**Pull request by grizzlytheodore**
## Description
PS design: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/970

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by grizzlytheodore** [comment]
@isra-fel  i can't quite figure out how to resolve the Ci failure, any insight?

**Comment by isra-fel** [comment]
Investigating

**Comment by isra-fel** [comment]
Previous CI failure was due to the target branch was not `generation`, which is used as an identifier of whether CI should run.
Now #16235 has been merged, I've changed the base branch back to `generation`, but there are a few conflicts.
Please resolve them, and the CI should be good. Thanks.

**Comment by grizzlytheodore** [comment]
thank you @isra-fel, i've resolved conflict. waiting for CI tests to pass now

**Comment by grizzlytheodore** [comment]
the failing CI is wrong: https://raw.githubusercontent.com/Azure/azure-powershell/cplat-generation-test/src/Compute/Compute.Autorest/docs/New-AzGalleryApplicationVersion.md

I've been getting this issue, not sure how to resolve

**Comment by isra-fel** [comment]
Something missing in the documents
```
 Target      : New-AzGalleryApplicationVersion.md
  Description : No synopsis found
  
  Target      : New-AzGalleryApplicationVersion.md
  Description : No description found
```

File: src/Compute/Compute.Autorest/custom/New-AzGalleryApplicationVersion.ps1 [hunk 1]
```diff
@@ -12,24 +12,22 @@ Create or update a gallery Application Version.
 .Description
 Create or update a gallery Application Version.
 #>
-
-
 function New-AzGalleryApplicationVersion {
```
Comment (isra-fel) [inline src/Compute/Compute.Autorest/custom/New-AzGalleryApplicationVersion.ps1]: PowerShell has strict format rules on comment-based documents, there should be no more than 2 empty lines between the comment and the function, so I removed them.

**Comment by grizzlytheodore** [comment]
@isra-fel  thank you. please merge

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**